plugins/MauticCitrixBundle/Command/SyncCommand.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/*
+ * @copyright   2016 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticCitrixBundle\Command;
+
+use MauticPlugin\MauticCitrixBundle\Entity\CitrixEventTypes;
+use MauticPlugin\MauticCitrixBundle\Helper\CitrixHelper;
+use MauticPlugin\MauticCitrixBundle\Helper\CitrixProducts;
+use MauticPlugin\MauticCitrixBundle\Model\CitrixModel;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI Command : Synchronizes registrant information from Citrix products.
+ *
+ * php app/console mautic:citrix:sync [--product=webinar|meeting|assist|training [--id=%productId%]]
+ */
+class SyncCommand extends ContainerAwareCommand
+{
+    /** @var CitrixModel */
+    protected $citrixModel;
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     */
+    protected function configure()
+    {
+        $this->setName('mautic:citrix:sync')
```

Comment:
I'm getting `[2016-11-24 02:31:55] mautic.ERROR: Warning: array_key_exists() expects parameter 2 to be array, integer given` when running this command.

This also syncs everybody who attended any goto meeting and not just those that are contacts in my database. Do we want that or do we want to match them with a contact ID. I'm wondering if contact ID should be stored in the plugin_citrix_events table as well?